### ISSUE TITLE
Blog about most active contributors

### ISSUE THREAD

**Issue by brianking** [mozilla/activate.mozilla.community#101]
Some Reps are doing an excellent job doing activate events. As a form of recognition, let's acknowledge them in a blog.

Perhaps not a blocker but it would be nice to have issue #66 Leaderboard in place first.

**Comment by brianking** [comment]
Waiting on #66 Leaderboard before writing and publishing this.

+ @comzeradd

**Comment by MichaelKohler** [comment]
Recognition will be something we will look at as part of our focus on Activate. We will be tracking this in the CD repository.
